doc/adr/0014-synchronous_peer_discovery.md

```diff
@@ -0,0 +1,73 @@
+# 14. Synchronous Peer Discovery
+
+Date: 2023-04-19
+
+## Status
+
+Proposed
```

Comment:
Please remember to change it to `Accepted` after approvals (we won't merge it with `Proposed` status). Or you can mark it as `Accepted` right now, and after approval we can immediately merge it.

Reply:
I will change it later.